workshop_app/send_mails.py

```diff
@@ -325,7 +375,7 @@ def send_email(	request, call_on,
 	else:
 		message = "Issue at Workshop Booking App please check"
 		try:
-			send_mail("Issue At Workshop Booking App Mailing", message, EMAIL_HOST_USER,
+			send_mail("Issue At Workshop Booking App Mailing", message, SENDER_EMAIL,
```

Comment:
the traceback email could be setup on the server itself. This shouldn't be a part of the code.